Search Attributes (#4)
temporalio/edu-interacting-with-workflows-typescript: search-attributes/README.md

# Querying Workflows with Search Attributes

![Temporal TypeScript SDK banner](../common/images/typescript-banner.png)

## 🗺️ What to Expect

> **In this chapter, you will learn how to use Search Attributes**

Author: Definitely should add this! Added

> **In this chapter, you will learn how to use Search Attributes to filter across your Workflows**

Reviewer: ```suggestion
**In this chapter, you will learn how to use Search Attributes to efficiently list and filter Workflows**
```

Specifically, by the end of this chapter, you will:

- Explain the difference between default and custom search attributes

> - Query your Workflows with Search Attributes with your CLI, your SDK, Web UI

Author: added

> - Query your Workflows with Search Attributes with your CLI, your SDK, and Web UI

Reviewer: ```suggestion
- List your Workflows, with filtering based on Search Attributes, using your CLI, your SDK, and Web UI
```

- Set custom search attributes with your CLI
- Set logic for search attributes in the Workflow and then query for them from your CLI or Client

### 💻 Lessons

- [What is the Temporal Visibility Feature?](what-is-the-visibility-feature.md)
- [What is a List Filter?](what-is-a-list-filter.md)
- [What is a Search Attribute?](what-is-a-search-attribute.md)
- [Custom Search Attributes](custom-search-attributes.md)
- [How to Create Custom Search Attributes](how-to-use-search-attributes.md)

> - [How to Use Custom Search Attributes](how-to-use-custom-search-attributes.md)

Reviewer 2: Duplicate links
